test/testFunctions.js

```diff
@@ -0,0 +1,59 @@
+import assert from 'assert';
+import { format } from 'util';
+import validator from '../src/index';
+
+function repeat(str, count) {
```

Comment:
suggestion: you can just use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat

Reply:
It seems that this function was added just before the release of Node 4.0.0 which added support for this. Seeing that we test on Node 6 and above I have updated this. Thanks for the suggestion!